aclk/README.md

```diff
@@ -110,3 +117,6 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
 Restart your Agent and [claim your node](/claim/README.md#how-to-claim-a-node).
 
 [![analytics](https://www.google-analytics.com/collect?v=1&aip=1&t=pageview&_s=1&ds=github&dr=https%3A%2F%2Fgithub.com%2Fnetdata%2Fnetdata&dl=https%3A%2F%2Fmy-netdata.io%2Fgithub%2Faclk%2FREADME&_u=MAC~&cid=5792dfd7-8dc4-476b-af31-da2fdb9f93d2&tid=UA-64295674-3)](<>)
+
+### ACLK Connection Details
```

Comment:
```suggestion
### ACLK connection details
```

Reply:
This heading is currently at the bottom of the page. I feel like it should go somewhere else, unless there's some other plan for it?